x/spec/types/spec.go

```diff
@@ -7,10 +7,14 @@ import (
 	"strings"
 	"unicode"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
 )
 
-const minCU = 1
+const (
+	minCU                = 1
```

Comment:
either start consts with capital or not, lets be consistant

Reply:
this is a weird comment... in go, a const that is exported to other libraries starts with a capital letter, a const that isn't is a cmall letter